LocalMultiplayerAgent/Startup.cs

```diff
@@ -36,10 +39,13 @@ public void ConfigureServices(IServiceCollection services)
                     options.SerializerSettings.NullValueHandling = NullValueHandling.Ignore;
                     options.SerializerSettings.Converters.Add(new StringEnumConverter());
                 });
+            
+            BasicAssetExtractor.Instance = new BasicAssetExtractor(SystemOperations.Default, Globals.MultiLogger);
```

Comment:
Is it preferable to add BasicAssetExtractor as singleton service here as the way Startup.cs in VmAgent implemented?

Reply:
I think this is fine, especially since we're inside the method 'ConfigureServices' so this shows the extractor being configured :)

Nit: Personally I think we may want to avoid using a singleton since a potential issue with using the singleton is there is no way to know if the singleton was initialized properly for the OS we want and it could lead to an unexpected error.